server.go

```diff
@@ -1,10 +1,11 @@
 package socketio
 
 import (
+	"log"
```

Comment:
log should be optional? there's nothing worse than a service that forces you to capture logs in a way you can't control